src/lib/mdns/minimal/records/ResourceRecord.h

```diff
@@ -62,6 +69,7 @@ class ResourceRecord
     QType mType;
     uint32_t mTtl = kDefaultTtl;
     FullQName mQName;
+    bool mCacheFlush = false;
```

Comment:
I am surprised to see the cache flush bit be added, but in such a way that the mDNS client does not invalidate any cached records if seen. Is this something that will come in a different change?

Reply:
the minmdns client is not a full mdns server so it has no "cache". Platform mdns would handle this clear outside of chip.

is this what you are referring to?

Reply:
The original intent of this change was to allow us to request cache flushes. It was missing in the general record, so I added it. The implementation we have currently doesn't support much with respect to caching. We maintain some info about commissionable nodes outside of the minimal implementation, but clear that on every new request regardless.